spec.bs

```diff
@@ -3065,17 +3058,21 @@ A <dfn>server auction response</dfn> is a [=struct=] with the following [=struct
   :: A [=list=] of interest group owner and interest group name pairs that bid
      in this auction.
   : <dfn>score</dfn>
-  :: {{double}} or Null. TODO
+  :: {{double}} or Null. For component auctions, contains the desirability

```

Comment:
let's link "desirability" to https://wicg.github.io/turtledove/#dom-scoreadoutput-desirability, if they're meant to be the same.

Same to other fields in this struct where a reference is avaiable, like interest group owner, name, etc,.